python/ray/tests/test_usage_stats.py

```diff
@@ -53,7 +54,11 @@
         "total_success": {"type": "integer"},
         "total_failed": {"type": "integer"},
         "seq_number": {"type": "integer"},
+        "extra_usage_tags": {"type": ["null", "object"]},
+        "total_num_nodes": {"type": ["null", "integer"]},
+        "total_num_running_jobs": {"type": ["null", "integer"]},
```

Comment:
```suggestion
        "total_running_jobs": {"type": ["null", "integer"]},
```

Reply:
I actually prefer the a bit longer name to be consistent with other `total_num_*` fields.